code401challenges/src/main/java/HashtableExercises/RepeatedWord.java

```diff
@@ -1,4 +1,35 @@
 package HashtableExercises;
 
 public class RepeatedWord {
+//    Input: Write a function that accepts a lengthy string parameter.
+    public static String repeatedWord(String lengthyStr){
+
+        //need a hashtable to store my lengthy string because lookup is constant O(1)
+        Hashtable duplicatesTbl = new Hashtable(30);
+
+        //Step 1: need to break up the string
+        //Step 2: account for spaces and special characters I.E .?$#$%^& etc.
+        String[] wordsArr = lengthyStr.trim().toLowerCase().split("\\W+");
+        String repeateWord = "";
+        int numOfOccurrences = 0;
```

Comment:
Since you're just figuring out if a word is already in the hashtable, you can just use booleans as the value: `duplicatesTbl.add(wordsArr[i], true)`

This is a good idea because `boolean`s use very little memory.